pkg/api/types.go

```diff
@@ -2154,3 +2154,38 @@ type RangeAllocation struct {
 	// a single allocated address (the fifth bit on CIDR 10.0.0.0/8 is 10.0.0.4).
 	Data []byte `json:"data"`
 }
+
+// LockSpec is the specification of a resource lock.
+type LockSpec struct {
+	// Who currently has the lock.
+	HeldBy string `json:"heldBy"`
+	// How long the lock is valid.
+	LeaseSeconds uint64 `json:"leaseSeconds"`
```

Comment:
Consistency.

It also used to be that uints weren't correctly represented in Swagger, and clients in some languages may not deal well with unsigneds.